src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -811,6 +811,7 @@ protected override void HandleInitializeRequestAsync(IRequestResponder<Initializ
             InitializeResponse initializeResponse = new InitializeResponse()
             {
                 SupportsConfigurationDoneRequest = true,
+                SupportsCompletionsRequest = m_engine is IDebugProgramDAP,
```

Comment:
This is going to be weird since we need to know which debug adapter we are targeting, for now we could always send `true` but it will not work in `lldb`.

Reply:
Is VS Code happy if we just return an empty array of completions?

Reply:
Yes. There will just be nothing for autocomplete, but the debug adapter does support the request, which will fail with Not Implemented exception